plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeModule.java

```diff
@@ -139,6 +140,8 @@ public void setup(Binder binder)
         // Azure
         logSynchronizerMapBinder.addBinding("abfs").to(AzureTransactionLogSynchronizer.class).in(Scopes.SINGLETON);
         logSynchronizerMapBinder.addBinding("abfss").to(AzureTransactionLogSynchronizer.class).in(Scopes.SINGLETON);
+        // GCS
+        logSynchronizerMapBinder.addBinding("gs").to(GcsTransactionLogSynchronizer.class).in(Scopes.SINGLETON);
```

Comment:
Adding the `GcsStorageFactory` binding separately in the `DeltaLakeGcsModule` was intentional.
The reasoning behind this choice was is that the `GcsStorageFactory` depends on `HiveGcsConfig` which comes with `HiveGcsModule`.

Reply:
`HiveGcsModule` is a real optional module, you can build Hive with out (i believe)
`DeltaLakeGcsModule` is just not an optional module, it's dependent by the main delta module. And since it encapsulates one class only, it's not really useful

it would be clean separation, if the `GcsTransactionLogSynchronizer` went into `DeltaLakeGcsModule` module
but i don't think this has any actual benefits